sources/Application/Instruments/MidiInstrument.h

```diff
@@ -60,6 +60,8 @@ class MidiInstrument : public I_Instrument {
   int remainingTicks_;
   bool playing_;
   bool retrig_;
+  bool porto_;
+  char porto_pending_[SONG_CHANNEL_COUNT];
```

Comment:
Why are we storing data for all channels in each instrument? (but not sharing the var between instruments)

Reply:
Ah good point! Thats supposed to be static, only 1 per all midi instruments, will fix! 👍🏻